migrator/postgreshelper/load.go

```diff
@@ -71,7 +72,8 @@ func Load(conf *config.Config, databaseName string) (*pgxpool.Pool, *gorm.DB, er
 					return err
 				}
 			}
-			log.WriteToStderrf("connect to gorm: %v", gormSource)
+			log.WriteToStderrf("connect to gorm: %v", strings.Replace(gormSource, adminConfig.ConnConfig.Password, "<REDACTED>", -1))
```

Comment:
cockroach has a redact package that looked interesting, but seemed like it might be a bit much to pull in for this since I could just do a simple replace, since strings.Replace returns an updated copy of the source string.